backend/src/main/java/wooteco/retrospective/dao/member/MemberDao.java

```diff
@@ -0,0 +1,75 @@
+package wooteco.retrospective.dao.member;
+
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+import wooteco.retrospective.domain.member.Member;
+
+import java.sql.PreparedStatement;
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public class MemberDao {
+
+    private final JdbcTemplate jdbcTemplate;
+
+    public MemberDao(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    private final RowMapper<Member> rowMapper = (rs, rn) ->
+            new Member(
+                    rs.getLong("id"),
+                    rs.getString("name")
+            );
```

Comment:
```suggestion
    private static final RowMapper<Member> ROW_MAPPER_OF_MEMBER = (rs, rn) ->
            new Member(
                    rs.getLong("id"),
                    rs.getString("name")
            );
```

상수로 바꾸는건 어떨까요? readOnly 로직이기 때문에 상수로 관리해도 임계영역 관련 이슈는 안날 것 같습니다.

Reply:
넵!